docs/resources/waf_domain.md

```diff
@@ -175,26 +202,42 @@ The following arguments are supported:
   + **$ssl_curves**
   + **$ssl_session_reused**
 
+* `traffic_mark` - (Optional, List) Specifies the traffic identifier.
+  WAF uses the configurations to identify the malicious client IP address (proxy mode) in the header,
+  session in the cookie, and user attribute in the parameter,
+  and then triggers the corresponding known attack source rules to block attack sources.
+  Only supports one traffic identifier.
+  The [traffic_mark](#Domain_traffic_mark) structure is documented below.
+
 * `timeout_settings` - (Optional, List) Specifies the timeout setting. Only supports one timeout setting.
   The [timeout_settings](#Domain_timeout_settings) structure is documented below.
 
+<a name="Domain_server"></a>
 The `server` block supports:
 
-* `client_protocol` - (Required, String) Protocol type of the client. The options include `HTTP` and `HTTPS`.
+* `client_protocol` - (Required, String) Protocol type of the client. The options include **HTTP** and **HTTPS**.
 
-* `server_protocol` - (Required, String) Protocol used by WAF to forward client requests to the server. The options
-  include `HTTP` and `HTTPS`.
+* `server_protocol` - (Required, String) Protocol used by WAF to forward client requests to the server.
+  The options include **HTTP** and **HTTPS**.
 
 * `address` - (Required, String) IP address or domain name of the web server that the client accesses. For example,
   `192.168.1.1` or `www.a.com`.
 
-* `port` - (Required, Int) Port number used by the web server. The value ranges from 0 to 65535, for example, 8080.
+* `port` - (Required, Int) Port number used by the web server. The value ranges from 0 to 65535, for example, **8080**.
+
+* `type` - (Required, String) Server network type. Valid values are: **ipv4** and **ipv6**.
+  If the value of this field is **ipv6**, then the corresponding server ip address should be of ipv6 type.
+  For example: **3ffe:1900:fe21:4545:0000:0000:0000:0000**.
+
+* `weight` - (Optional, Int) The load balancing algorithm will assign requests to the origin
+  site according to this weight. It`s a redundant field in cloud mode.
+  Defaults to **1**.
```

Comment:
* `weight` - (Optional, Int) The load balancing algorithm will assign requests to the origin
  site according to this weight.
  Defaults to **1**.